pyracing/response_objects/iracing_data.py

```diff
@@ -2,7 +2,7 @@
 
 
 # I believe this maps what I would call a series, but iRacing calls this a season, so I want to be consistent
-class Season:
+class CurrentSeasons:
```

Comment:
This model just represents a single season, and it does not necessarily _have_ to be current. I would say this makes more sense being just called `Season`